### ISSUE TITLE
Add an explicit error for return code -9

### ISSUE THREAD

**Issue by ebolyen** [qiime2/q2-dada2#86]
This seems to be always be a SIGKILL event which is almost *always* because of an OOM error. It is probably worth just catching and raising an error about SIGKILL/Memory when it happens.

**Comment by benjjneb** [comment]
Why are these OOM errors happening so much in the Q2 plugin? We have rarely gotten such reports on the R side. I think the only times I remember it happening are when people were using 32-bit R.

**Comment by ebolyen** [comment]
I suspect a lot of people are using VirtualBox, which needs to be configured ahead of time to have an even remotely reasonable amount of memory.

**Comment by benjjneb** [comment]
Ah, that makes sense.

So to implement this, the -9 exit code would need to be caught from Python where it makes the Rscript call?

**Comment by ebolyen** [comment]
Yup, just like this case: https://github.com/qiime2/q2-dada2/blob/master/q2_dada2/_denoise.py#L128-L133
